.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - '6'
+cache:
+  directories:
+    - node_modules
```

Comment:
How's the cache invalidated?

Reply:
See https://docs.travis-ci.com/user/caching/#How-does-caching-work%3F

Reply:
Probably similar to CircleCI and others who create checksums.

Reply:
`node_modules` is roughly 150mb and the [link above](https://docs.travis-ci.com/user/caching/#How-does-caching-work%3F) says:

> If you store archives larger than a few hundred megabytes in the cache, it is unlikely that you’ll see a big speed improvement.

Not sure what that means, but let's test if we can get a meaningful change in build time. It probably will.

Reply:
You can easily test this by triggering a new build with an empty commit.